myhoard/util.py

```diff
@@ -366,10 +366,10 @@ def are_gtids_in_executed_set(gtid_executed, ranges, *, exclude_uuid=None):
 
 
 @contextlib.contextmanager
-def mysql_connection(*, ca_file=None, db="mysql", host="127.0.0.1", password, port, timeout=DEFAULT_MYSQL_TIMEOUT, user):
-    ssl = None
-    if ca_file:
-        ssl = {"ca": ca_file}
+def mysql_connection(
+    *, ca_file=None, db="mysql", host="127.0.0.1", password, port, timeout=DEFAULT_MYSQL_TIMEOUT, user, require_ssl=False
+):
+    ssl = {"ca": ca_file} if ca_file else {"require": True} if require_ssl else None
```

Comment:
If for any reason it does not enable SSL there is another possibility to use `ssl_ca` parameter for the connection and it will enable SSL (check the code I've posted)

Reply:
We need to pass it a truthy value because of this line: https://github.com/PyMySQL/PyMySQL/blob/885841f3fee416c222a75d83a81f74d3dcd71b51/pymysql/connections.py#L279

Reply:
> If for any reason it does not enable SSL there is another possibility to use `ssl_ca` parameter for the connection and it will enable SSL (check the code by the link I've posted)

My understanding is that we want to support enabling SSL without passing `ca_file`. Or I am missing something?